<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDlg</class>
 <widget class="QDialog" name="StartDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Page</string>
  </property>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sexLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="birthdateLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Birthdate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="sexCombo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="traitLbl">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a Trait:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="traitCombo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="nameTxt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="monthCombo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="dayCombo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="yearCombo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
